controller/db/controller_store.go

```diff
@@ -0,0 +1,109 @@
+/*
```

Comment:
I'm a little hesitant to store controller information in the db, especially the 'isOnline' and 'lastJoined'. Won't the different controllers have different opinions on that, depending on who is connected to who and when?

Reply:
Ah, i got further and saw that this is gated to the leader, so I'm less concerned :+1: